charts/newrelic-infra-operator/templates/clusterrole.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "newrelic-infra-operator.fullname" . }}
+  labels:
+    {{- include "newrelic-infra-operator.labels" . | nindent 4 }}
+rules:
+  - apiGroups: ["rbac.authorization.k8s.io"]
+    resources: ["clusterrolebindings"]
```

Comment:
In case we want to create automatically secrets in user namespace, we would need as well get, update and delete (for garbage collect).

Otherwise the user should create it manually or we should mount the license is a plain env var

Reply:
I think it's reasonable to add a secret permissions right now.

Reply:
In the past we've had users quite reticent to grant `get` permissions to secretsa cross the cluster. Would this be doable with just `create` and `update` and `delete`?